lightly/loss/symmetrized_loss.py

```diff
@@ -0,0 +1,85 @@
+import torch
+import numpy as np
+
+class SymmetrizedLoss(torch.nn.Module):
+    """Implementation of the Symmetrized Loss.
```

Comment:
I'd suggest to give this loss a more descriptive name (for example `SymmetricNegCosineSimilarityLoss` or `NegCosineSimilarityLoss`).